src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.html

```diff
@@ -125,42 +128,37 @@ <h1 class="govuk-heading-s">Hearing dates</h1>
                 class="govuk-link">Update</a>
             </dd>
           </div>
-          <div class="govuk-summary-list__row">
+          <div class="govuk-summary-list__row" *ngIf="!actualHearingDay.notRequired">
             <dt class="govuk-summary-list__key">Names</dt>
             <dd class="govuk-summary-list__value">{{getPartiesNames(actualHearingDay)}}</dd>
             <dd class="govuk-summary-list__actions"></dd>
           </div>
-          <div class="govuk-summary-list__row">
+          <div class="govuk-summary-list__row" *ngIf="!actualHearingDay.notRequired">
             <dt class="govuk-summary-list__key">How participants will attend</dt>
             <dd class="govuk-summary-list__value">{{getPartiesAttendenceMethod(actualHearingDay)}}</dd>
             <dd class="govuk-summary-list__actions"></dd>
           </div>
           <div class="govuk-summary-list__row" *ngFor="let party of actualHearingDay.actualDayParties; let i = index;">
-            <dt class="govuk-summary-list__key">Attendee {{i + 1}}</dt>
-            <dd class="govuk-summary-list__value">
+            <dt class="govuk-summary-list__key" *ngIf="!actualHearingDay.notRequired">Attendee {{i + 1}}</dt>
+            <dd class="govuk-summary-list__value" *ngIf="!actualHearingDay.notRequired">
               Role: {{party.partyRole}}<br>
               First name: {{party.individualDetails.firstName}}<br>
               Last name: {{party.individualDetails.lastName}}<br>
               Organisation: {{party.actualOrganisationName}}<br>
-              Attendee method: {{party.partyChannelSubType}} <br>
-              Attendee representing: {{getRepresentingAttendee(party.representedParty)}}<br>
+              Attendance Type: {{party.partyChannelSubType}} <br>
+              Attendance representing: {{getRepresentingAttendee(party.representedParty)}}<br>
```

Comment:
could this *ngIf="!actualHearingDay.notRequired" extracted to 1 div ngIf rather than tens of ngIf judgement?

Reply:
Done